stream-chat-swift-docs/docs/guides/working-with-messages.md

```diff
@@ -274,10 +277,11 @@ There are two ways of replying a message:
   
     Initiating a thread reply takes a user into thread details screen and the resulting message will look like a normal message that is placed inside the thread. It is also possible to duplicate it to the parent channel.
   
- <img src="../img/thread-reply.png" width="20%" />
+    <img src="../img/thread-details.png" width="20%" />
```

Comment:
It'd be better if the original message belonged to another user

Reply:
Ok let's change that